client/src/components/commonComponents/NavBar/index.js

```diff
@@ -1,22 +1,62 @@
 import React from 'react';
-import { List, ListItem, Typography, Grid } from '@material-ui/core';
+import { List, ListItem, Typography, Grid, Box, Link } from '@material-ui/core';
+import SearchIcon from '@material-ui/icons/Search';
+import MenuIcon from '@material-ui/icons/Menu';
+import ShoppingCartOutlinedIcon from '@material-ui/icons/ShoppingCartOutlined';
 import logo from './logo.svg';
+import blog from './downearthblog.webp';
+
```

Comment:
Because we have several images put them inside dir and name it **(assets)**

Reply:
done :+1: